companion/lib/Surface/USB/ElgatoStreamDeck.ts

```diff
@@ -63,6 +64,17 @@ function getConfigFields(streamDeck: StreamDeck): CompanionSurfaceConfigField[]
 	return fields
 }
 
+/**
+ * The latest firmware versions for the SDS at the time this was last updated
+ */
+const LATEST_SDS_FIRMWARE_VERSIONS: Record<string, string> = {
+	AP2: '1.05.009',
+	ENCODER_AP2: '1.01.012',
+	ENCODER_LD: '1.01.006',
+}
+const SDS_UPDATE_TOOL_URL = 'https://bitfocus.io/?elgato-sds-firmware-updater'
+const SDS_UPDATE_VERSIONS_URL = 'https://builds.julusian.dev/builds/sds-test.json'
```

Comment:
Would it make more sense to serve this from GitHub raw like so?:
https://raw.githubusercontent.com/bitfocus/companion/refs/heads/main/nodejs-versions.json

Reply:
Both of these urls are placeholders currently, waiting on the update tool to be available